src/test/resources/concordion/mockserver/Overview.html

```diff
@@ -0,0 +1,17 @@
+<html xmlns:c="http://www.concordion.org/2007/concordion">
+<body>
+<h1>Overview</h1>
+<p>
+    This is <b>UNOFFICIAL</b> MockServer Concordion extension. Concordion MockServer extension can be used for creating expectation for <a href="http://www.mock-server.com">MockServer</a> in concordion specification file.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)